test/pages/UnderpaymentTypePageSpec.scala

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package pages
+import base.SpecBase
+import models.{UnderpaymentAmount, UnderpaymentType, UserAnswers}
+
+class UnderpaymentTypePageSpec extends SpecBase {
+
+  "UnderpaymentTypePageSpec" must {
+
+    "remove deselected Underpayments" in {
+      val answers: UserAnswers = UserAnswers("some-cred-id")
+        .set(UnderpaymentTypePage, UnderpaymentType(customsDuty = false, importVAT = false, exciseDuty = false)).success.value
+        .set(CustomsDutyPage, UnderpaymentAmount(111, 222)).success.value
```

Comment:
these UnderpaymentAmount(111, 222), UnderpaymentAmount(333, 444), UnderpaymentAmount(555, 666) are duplicated in this file, could they not be replaced with UnderpaymentSummaryData.cdUnderpayment and the other two from UnderpaymentSummaryData

Reply:
done